src/components/Pages/TextPictureCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface Props {
+  title: string;
+  text: string;
+  imgPath: URL;
+}
+
+const TextPictureCard = ({ title, text, imgPath }: Props): JSX.Element => {
+  return (
+    <div className="flex w-full mx-auto justify-between flex-wrap mt-10">
+      <div className="max-w-6xl  m-auto">
+        <h1 className="text-center text-2xl">{title}</h1>
+        <p className="max-w-lg text-xl mt-3 px-3">{text}</p>
+      </div>
+      <img
+        src={imgPath.href}
+        alt="Vektormorro"
```

Comment:
The alt should probably be extracted and put in `ForForeldre.ts` instead, so that each image has it's own unique alt.